examples/timelock/TIMELOCK_01.sol

```diff
@@ -19,6 +21,16 @@ contract TIMELOCK_01 is TimelockProposal {
         return "Timelock proposal mock";
     }
 
+    /// @notice always reverts, do not use this method for timelock proposals
+    function getCalldata()
+        public
+        view
+        override(Proposal, TimelockProposal)
+        returns (bytes memory data)
+    {
+        return TimelockProposal.getCalldata();
```

Comment:
what is the point of overriding here?

Reply:
would it be better to implement AlphaProposal in the Proposal so we don't need to have 2 parent classes?